crates/katana/executor/src/implementation/blockifier/mod.rs

```diff
@@ -26,12 +26,6 @@ use crate::{
 
 pub(crate) const LOG_TARGET: &str = "katana::executor::blockifier";
 
-// TODO: @kariy Which value should be considered here? I took the default
-// value from the previous implementation.
-// Previous: https://github.com/dojoengine/blockifier/blob/7459891173b64b148a7ce870c0b1d5907af15b8d/crates/blockifier/src/state/cached_state.rs#L731
-// New code: https://github.com/starkware-libs/blockifier/blob/a6200402ab635d8a8e175f7f135be5914c960007/crates/blockifier/src/state/global_cache.rs#L17C11-L17C46
-pub(crate) const CACHE_SIZE: usize = 100;
```

Comment:
Curious, is this `CACHE_SIZE` something that we want to customize at some point, or we shouldn't care more about it?

Reply:
unsure but they have removed the `GlobalContractCache` when creating a new `CachedState` which what we used `CACHE_SIZE` before